TTS/tts/utils/text/phonemizers/__init__.py

```diff
@@ -30,6 +31,7 @@
 DEF_LANG_TO_PHONEMIZER["zh-cn"] = ZH_CN_Phonemizer.name()
 DEF_LANG_TO_PHONEMIZER["ko-kr"] = KO_KR_Phonemizer.name()
 DEF_LANG_TO_PHONEMIZER["bn"] = BN_Phonemizer.name()
+DEF_LANG_TO_PHONEMIZER["be"] = BEL_Phonemizer.name()

```

Comment:
it is better not to default it to your implementation. Espeak also supports BE. Since it doesn't require Java, I think it is better to default to that.

Maybe you can key your implementation with a different keyword like `be_fanetyka`

Reply:
We should also handle the case when java backend is not installed like here -> https://github.com/coqui-ai/TTS/pull/2843

Reply:
Espeak has very bad phonemizer for Belarusian language - no many rules, no stresses, no changes on the border of words. It's not really useful. Since CoquiTTS doesn't have espeak phonemizer for Belarusian language, I don't think it will be good to add it.
I changed TTS/tts/utils/text/belarusian/phonemizer.py for support case when no java installed. Belarusian phonemizer initialized only on the first real call, so, it will not affect on other phonemizers and will not load any java code before real usage.